Sources/hostmgr/commands/vm/VMFetch.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
```

Comment:
This import is not necessary (I verified it locally).

```suggestion
```

Reply:
Thanks!! I do this a lot 😅